scripts/modules/b2b-account/edit-quote.js

```diff
@@ -1411,6 +1412,9 @@ define([
             }
 
             return false;
+        },
+        convertLocalDate: function() {
```

Comment:
Can we create this method like
convertLocalDate: function(dateToConvert) {
         if(dateToConvert)
            {
              return new Date(dateToConvert).toLocaleDateString();
            }
           return "";
}

if possible to do the above changes, update all references on the edit and view-quote page

Reply:
No need to pass arguments here because we are getting span value during DOM load.